src/main/java/frc/robot/Constants.java

```diff
@@ -26,6 +27,13 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+  public record RobotSetup(String name, String canivore_name, GyroType gyroType) {}
```

Comment:
Other things we could add here are wheelbase, or other physical robot setup values.

Reply:
We'll do this later because we have to make photonvision, i'll add this as an issue.